source/lambda/requirements.txt

```diff
@@ -6,20 +6,21 @@ aws_lambda_typing==2.18.0
 #sub-dependencies
 # NONE
 
-boto3==1.28.52
+boto3==1.28.66
 #sub-dependencies
-botocore==1.31.52
+botocore==1.31.66
 jmespath==1.0.1
 python-dateutil==2.8.2
-s3transfer==0.6.2
+s3transfer==0.7.0
 six==1.16.0
-urllib3==1.26.16
+urllib3==2.0.7
```

Comment:
ERROR: Cannot install -r requirements.txt (line 11) and urllib3==2.0.7 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested urllib3==2.0.7
    The user requested urllib3==2.0.7
    botocore 1.31.66 depends on urllib3<1.27 and >=1.25.4; python_version < "3.10"